docs/source/development/defining_new_models.md

```diff
@@ -311,10 +314,9 @@ def from_config(cls, config: dict[str, Any]) -> FreshWaterModel:
         raw_interval = pint.Quantity(config["freshwater"]["model_time_step"]).to(
             "minutes"
         )
-        # Round raw time interval to nearest minute
-        update_interval = timedelta64(int(round(raw_interval.magnitude)), "m")
-        start_time = datetime64(config["core"]["timing"]["start_time"])
-        no_of_pools = config["freshwater"]["no_of_pools"]
+        start_date, update_interval = extract_model_time_details(
+                config, cls.model_name
+            )
     # Catch cases where Values or dimensions are wrong
```

Comment:
Was `no_of_pools` meant to go? It is an example of a model specific config check? If we do switch to a pattern of separating the timing checks off in their own `try` (see other comments), then having an example of then checking model specific bits would be good.

Reply:
Nope, was not meant to go!